debian/changelog

```diff
@@ -1,3 +1,155 @@
+ostree (2018.4-0) master; urgency=medium
+
+  * Resync Debian packaging changes (T22022):
+    - Drop package version back to 0 to appease jenkins
```

Comment:
Couldn’t we version it `2018.4-1endless0`? Having decrementing version numbers in the `debian/changelog` seems a bit wrong.

Reply:
Sure. `jenkins-obs-build` will start the packaging version at 0, which is why I have that here. However, since we have patches on the main branch, the upstream version will move past this, so it won't matter. See https://phabricator.endlessm.com/w/software/build/ci-integration/#version-numbering.